Common_Words_Scraping.py

```diff
@@ -0,0 +1,45 @@
+# coding: utf-8
```

Comment:
not needed in python3 see: https://stackoverflow.com/questions/14083111/should-i-use-encoding-declaration-in-python3